### PR TITLE
SW-3858 map view for planting site details page

### PR DESCRIPTION
**Pull request by karthikbtf**
- generate map data from aggregated info
- no popups yet, separate PR

Terraware App 2023-07-11 14-54-18

### CODE REVIEW

**Review by constanzauanini — APPROVED** [review]
Looks great!

**Comment by karthikbtf** [comment]
> Looks great!

Thanks @constanzauanini - there's a parent PR for this https://github.com/terraware/terraware-web/pull/1541 if I can get eyes on it too.

**Comment by karthikbtf** [comment]
> > Looks great!
> 
> Thanks @constanzauanini - there's a parent PR for this #1541 if I can get eyes on it too.

Added an update to reuse code.

File: src/services/MapService.ts [hunk 1]
```diff
@@ -356,6 +314,23 @@ const getMapDataFromAggregation = (plantingSite: PlantingSiteAggregation): MapDa
   };
 };
 
+// private util
+const getMonitoringPlotMapData = (
```
Comment (constanzauanini) [inline src/services/MapService.ts:318]: 💯

**Review by constanzauanini — APPROVED** [review]

**Review by charlie-tf — APPROVED** [review]
This looks great, Karthik!

**Pull request merged**